### PR TITLE
my ec2 file

### PR DESCRIPTION
**Pull request by alyssahoward**

### CODE REVIEW

**Review by alyssahoward — COMMENTED** [review]
oops

**Review by snipesa — APPROVED** [review]

File: ec2.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Set up a Boto3 EC2 client
```
Comment (relindiss1) [inline ec2.tf]: this is python instead of terraform

File: ec2.tf [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#resource "aws_instance" "test" {
```
Comment (emekanwa) [inline ec2.tf:1]: Please take out the # sign

Comment (alyssahoward) [reply ec2.tf:1]: ok

**Pull request closed without merging**